plugin/parser/differ/mysql/differ.go

```diff
@@ -56,9 +62,11 @@ func (*SchemaDiffer) SchemaDiff(oldStmt, newStmt string) (string, error) {
 				diff = append(diff, &stmt)
 				continue
 			}
-
+			indexMap := buildIndexMap(oldNodes, newStmt.Table.Name)
```

Comment:
This `buildIndexMap` looks unnecessary. We can use `oldTableMap` to get the corresponding statement.